embedded-graphics/src/image/image_bmp.rs

```diff
@@ -129,17 +131,14 @@ where
                 return None;
             }
 
-            let offset = ((h - 1 - y) * w) + x;
+            let offset = (((h - 1 - y) * w) + x) as usize;
```

Comment:
hello! I was testing out this branch and noticed this calculation (and the former revision) don't take stride into account, so BMP images that have a width that is not a multiple of 4 don't display properly

Reply:
Hi, thanks for spotting this! As the bug is present in the currently released version, can you open a separate issue describing it please? This PR is already pretty long :slightly_smiling_face:. It would be great if you could also include a failing test case and/or a sample image to test with.